crates/fluvio-hub-util/src/packagemeta.rs

```diff
@@ -46,6 +46,13 @@ impl Default for PackageMeta {
 }
 
 impl PackageMeta {
+    /// Retrieves the fully qualified name for the `PackageMeta`.
+    ///
+    /// Eg: `infinyon-example-0.0.1`
+    pub fn id(&self) -> String {
+        format!("{}-{}-{}", self.group, self.name, self.version)
```

Comment:
where is this used?
I see that we currently have `packagename_transform`?

Reply:
Updating approach!